challenge.js

```diff
@@ -9,6 +9,8 @@
  */
 
 // ANSWER
+Using a javascript page embedded in the HTML. The javascript could invoke functions from jquery that listen to the page through events and applies handlers to them.
```

Comment:
That's absolutely relevant to this topic. We frequently alter or attach new HTML elements to the DOM in an event handler to create exciting, interactive web pages. 1/2